tasks/npsp_delete.py

```diff
@@ -0,0 +1,117 @@
+### Delete this file when CCI supports deletes with where (i.e. Nov 2019)
```

Comment:
This was included in cumulusci 2.5.8 (looks like we missed it in the release notes)

Reply:
This file is no longer used.